ai-24sp/prototypes/prototype-00/run_model.py

```diff
@@ -0,0 +1,14 @@
+from loader import load_images, load_labels, save_single_image
+from network import Network
+
+nn = Network.fromPBJSON("model-30epochs-10batch-1_0eta.pbjson")
+
```

Comment:
current `train.py` outputs to the file name `model.pbjson` maybe this script could be modified to take a file name as a command line argument.